governance_artifacts/zk/gcir_harness.py

```
@@ -0,0 +1,142 @@
+    except Exception as e:  # pragma: no cover - fallback only
+        print(f"[harness] PyYAML unavailable or parse error ({e}); aborting", file=sys.stderr)
+        sys.exit(3)
+
+
+def run_rego(decision: dict) -> bool:
+    """Return True if the Rego policy says `allow` (compliant)."""
+    inp = {"decision": decision}
+    with tempfile.NamedTemporaryFile("w", suffix=".json", delete=False) as f:
+        json.dump(inp, f)
+        inp_path = f.name
+    out = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)